packages/statistics-module/src/links/domain/Statistic.js

```diff
@@ -0,0 +1,7 @@
+// TODO: create the implementation of statistic with the next structure
+
+// id       -> id of table
+// linkId   -> id of link
+// userId   -> id of user
```

Comment:
Based on the requirements that we have specified in the document in Notion with the business rules, the schema is too simplistic for the next reasons:
* How are we gonna handle the metadata(IP address, browser, and so on) of the resource? 
* How can we differentiate the statistics of users and links?
* How are we gonna handle the statistics for links that contain UTM's and belongs to a campaign?
* What are the metrics that we gonna handle as a statistics unit?

Reply:
We are going to add those, but we are going to do it in a following ticket, this was only the starting point